Makefile

```diff
@@ -58,7 +58,7 @@ lint: env/pyvenv.cfg
 		black --check $(ALL_PY_SRCS) && \
 		isort --check $(ALL_PY_SRCS) && \
 		ruff $(ALL_PY_SRCS) && \
-		mypy $(PY_MODULE) && \
+		mypy --sqlite-cache $(PY_MODULE) && \
```

Comment:
Not directly related to this changeset, but this should be significantly faster than mypy's default cache.